docs/getting_started.md

```diff
@@ -10,9 +10,15 @@
 
 It is recommended that you use a python virtual environment to install the necessary dependencies
 
-1. Setup a python 3.7 virtual environment. This is actually quite easy to do now.
-    * `python3.7 -m venv <path/to/venv>`
-    * `source <path/to/venv>/bin/activate`
+1. Setup a python 3.7 virtual environment. This is actually quite easy to do
+    now.
+
+> Use hidden `.venv` or normal `venv` folder for virtual env as we are
+> ignoring this in flake8 configurtation in tox.
```

Comment:
Sorry for syntax nitpicking, but it would help if this block of text were aligned with the previous text, removing the blockquote marks. That way, the following list would not be considered to be a code block.

Reply:
Wanted to emphasize this by blockquote but I don't mind to let it as part of the previous text